RecoTracker/LSTCore/standalone/README.md

```diff
@@ -53,6 +62,12 @@ source setup.sh # if on UCSD or Cornell
 source setup_hpg.sh # if on Florida
 ```
 
+Currently, the data files for LST need to be copied manually. This is done by running:
+
+```bash
+git clone git@github.com:cms-data/RecoTracker-LSTCore.git data
```

Comment:
It's probably worth mentioning where this `data` directory should reside. It's not clear whether it's in `standalone` or `LSTCore`.

Reply:
You are right, and in fact I had the directory wrong in one case. I now fixed that and added an explicit mention of the directory where the data files should end up.

Reply:
Looking at the `setup.sh`, after it's called the `$CMSSW_SEARCH_PATH` will already be set.
So, the same `git clone git@github.com:cms-data/RecoTracker-LSTCore.git $CMSSW_BASE/external/$SCRAM_ARCH/data/RecoTracker/LSTCore/data` should work (note that cd in and out of the target directory is not really necessary)

Reply:
oh, nevermind; that search path will not include the local CMSSW checkout